### ISSUE TITLE
About BSN improvement B

### ISSUE THREAD

**Issue by VSunN** [wzmsltw/BSN-boundary-sensitive-network#20]
Hi,
Mentioned in your ActivityNet report improvement B that you adopt TSN and P3D pretrained on Kinetics-400 dataset for video feature extraction. I wonder whether the model fine-tuned on the ActivityNet dataset. If not, are you using the feature of the last pooling layer?

**Comment by wzmsltw** [comment]
Hi @VSunN , the model is not fine-tuned. And we use the classification layer（without softmax）as feature.

**Comment by VSunN** [comment]
Thank you for your reply.
Does it mean that for the P3D network, when the input interval is 16 frames, the output feature length is 400 instead of 200?
@wzmsltw

**Comment by wzmsltw** [comment]
Yes
